serving/src/main/java/ai/djl/serving/ModelServer.java

```diff
@@ -265,14 +265,17 @@ private void initModelStore() throws IOException {
         Set<String> startupModels = ModelManager.getInstance().getStartupModels();
 
         String loadModels = configManager.getLoadModels();
+        Path modelStore = configManager.getModelStore();
         if (loadModels == null || loadModels.isEmpty()) {
-            return;
+            if (modelStore != null) {
```

Comment:
There should be a way to disable this too. Maybe null for "all" and empty string for loading no models